test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -232,6 +232,7 @@ def setUp(self):
     jobnum=${PBS_JOBID%%.*}
     base="$cpuset_base/pbspro.slice/pbspro-${jobnum}.*.slice"
 fi
+echo "cpuset base path is $base"
```

Comment:
This looks like shell syntax, not Python syntax.

Reply:
@mike0042 this is a shell script because.

Reply:
Sorry, I needed more lines for context. I see now.